Library/Homebrew/compilers.rb

```diff
@@ -1,7 +1,7 @@
 # @private
 module CompilerConstants
-  GNU_GCC_VERSIONS = %w[4.3 4.4 4.5 4.6 4.7 4.8 4.9 5 6 7].freeze
-  GNU_GCC_REGEXP = /^gcc-(4\.[3-9]|[5-7])$/
+  GNU_GCC_VERSIONS = %w[4.6 4.7 4.8 4.9 5 6 7].freeze
+  GNU_GCC_REGEXP = /^gcc-(4\.[6-9]|[5-7])$/
```

Comment:
Let's leave these as-is unless they break something as people may still want to use their own versions of the old formulae.